modules/KalturaSupport/resources/mw.KCuePoints.js

```diff
@@ -236,16 +238,19 @@
 				var _this = this;
 
 				var thumbNewCuePoints = [];
-				var codeNewCuePoints = [];
+				var managerNewCuePoints = [];
 				//Only add new cuepoints
 				$.each(rawCuePoints, function (id, rawCuePoint) {
 					if (!_this.associativeCuePoints[rawCuePoint.id]) {
 						_this.associativeCuePoints[rawCuePoint.id] = rawCuePoint;
 
-						if (rawCuePoint.cuePointType === 'codeCuePoint.Code')
+						if (rawCuePoint.cuePointType === 'codeCuePoint.Code' || rawCuePoint.cuePointType === 'thumbCuePoint.Thumb')
```

Comment:
@esakal the new manager cuepoints type is a composite type which holds both thumb and code cuepoints and it serves only one use case which is not core.
I would like to avoid such coupling in core components as possible.